### PR TITLE
fix web_application_routing is not enabled in this cluster

### PR DESCRIPTION
**Pull request by tg123**
---

This checklist is used to make sure that common guidelines for a pull request are followed.

### Related command

az aks addon update --addon web_application_routing .....

### General Guidelines

- [ ] Have you run `azdev style <YOUR_EXT>` locally? (`pip install azdev` required)
- [x] Have you run `python scripts/ci/test_index.py -q` locally?

For new extensions:

- [ ] My extension description/summary conforms to the [Extension Summary Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/extensions/extension_summary_guidelines.md).

### About Extension Publish

There is a pipeline to automatically build, upload and publish extension wheels.  
Once your pull request is merged into main branch, a new pull request will be created to update `src/index.json` automatically.  
You only need to update the version information in file setup.py and historical information in file HISTORY.rst in your PR but do not modify `src/index.json`.

### CODE REVIEW

**Comment by yonzhan** [comment]
Thank you for your contribution! We will review the pull request and get back to you soon.

File: src/aks-preview/azext_aks_preview/custom.py [hunk 1]
```diff
@@ -1512,7 +1512,7 @@ def aks_addon_update(cmd, client, resource_group_name, name, addon, workspace_re
         if (instance.ingress_profile is None) or (instance.ingress_profile.web_app_routing is None) or not instance.ingress_profile.web_app_routing.enabled:
             raise InvalidArgumentValueError(f'Addon "{addon}" is not enabled in this cluster.')
 
-    if addon == "monitoring" and enable_msi_auth_for_monitoring is None:
+    elif addon == "monitoring" and enable_msi_auth_for_monitoring is None:
```
Comment (FumingZhang) [inline src/aks-preview/azext_aks_preview/custom.py:1515]: @tg123, what's the problem you are trying to resolve? If you want to enable the Web Application Routing add-on on an existing cluster, please follow [doc](https://learn.microsoft.com/en-us/azure/aks/web-app-routing?tabs=without-osm) and use command `az aks enable-addons` instead.

Comment (tg123) [reply src/aks-preview/azext_aks_preview/custom.py:1515]: fix regression introduced by #6311 

original code 
```
if addon == "web_application_routing":
else:
   xxx
```

after 6311

```
if addon == "web_application_routing":

if addon == "monitoring":
else:
   xxx
```

which means when `addon == web_application_routing`, xxx will be always executed and raise ex

Comment (FumingZhang) [reply src/aks-preview/azext_aks_preview/custom.py:1515]: OK, got it

**Comment by FumingZhang** [comment]
The issue has been fixed by PR #6512

**Pull request closed without merging**